### PR TITLE
Integrate Facebook Ads into your Android App in Android Studio

### LINKED ISSUE #4585: [API] Integrating Facebook Ads with your Android Application in Android studio
#### Introduction
Besides developing mobile applications for technological adjustments, developers always look towards getting something in return. One of the greatest ways of earning from applications being advertisement, this article shows developers how they can integrate banner and interstitial ads in their application.

With Facebook Audience Network, developers are allowed to monetize their android applications with Facebook Ads. This motivates developers to come up with new applications every now and then.

#### Key takeaways 
- Overview of all the requirements
- Set up your Android application project
- Installation of the required **sdk** i.e **(Facebook Network Audience) sdk** 
       and other requirements.
- Steps of **Integrating**FAN sdk and adapter into your Android application

#### Article quality
The official documentation gives a general directive using the initial generated code, which sometimes proves hard to follow.
I intend to give a clear step by step explanation with more understanding simplicity and flexibility with different versions (not offered in the available documentation).

#### References
N/A

### Templates to use as guides
- [How To Guide Template](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/how-to-guide-template.md)
- [Software Review Template](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/software-review-template.md)
- [Tutorial Template](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/tutorial-template.md)

### PR DESCRIPTION
**Pull request by del-ui**
### Article checklist

> If you have not already, please go over our [Resources Page](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/resources-page.md) for more tips and tricks on how to improve your overall technical writing so reviews are swifter, increase the chance of a payout and provide more value to our readers.

> For first-time contributors and for more details on our submission guidelines, **see our [Contributing Guidelines](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/CONTRIBUTING.md)**.

#### Formatting and structure checks
- [x] Have you used the correct folder and file structure? - **see [Contribution Guidelines](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/CONTRIBUTING.md)**
- [x] Is your article properly formatted in Markdown? - **see [Github Markdown guide](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet)**
- [x] Have you used our preferred heading structure? - **[H3 (###) and up](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#headers)**
- [x] Does your article follow any of the suggested structure formats? - **see [example formats](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md#example-format-structure)**

- [x] Is your article over 750 words? - **Tool**: [Wordcounter.net](https://wordcounter.net)
- [x] Is the article you are submitting an in-depth and unique article? Does it go beyond what is in the official docs and what is covered in other blog sites. See these articles as [examples](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/resources-page.md).
- [x] Does your article provide enough value and detail about your topic? - **Articles should be clear, accurate, and fully explained.**
- [x] Can your article be understood by beginners? - **Assume the audience is smart but has no prior exposure to the common terminology in your article.**

- [x] Have you included a hero image that is 600x400 pixels and have the copyright to use it? - **Tools:** [Pixlr Image Editor](https://pixlr.com/e) for resizing and [Unsplash](https://unsplash.com) for Creative Commons images
      
#### Grammar & plagiarism checks
- [x] Have you spellchecked and grammar checked your article? - **Tools:** [Write&Improve](https://writeandimprove.com/?lang=en-GB) - [ProWritingAid](https://prowritingaid.com/) - [Grammarly](https://grammarly.com) 
- [x] Please place your article through a 3rd party plagiarism checker. We suggest using Quetext, [this tool](https://www.quetext.com/) is free and has a daily limit. **We typically accept articles with 10% or less.**
- [x] Have you checked your article for readability? - **Tool:** [Hemmingway](http://www.hemingwayapp.com)
- [ ] Have you added sources for quotes and images that aren't yours?

#### Technical checks
- [x] Are your code snippets properly formatted for syntax highlighting - **see [Syntax guide](https://gohugo.io/content-management/syntax-highlighting)**
- [x] Have you checked your code runs correctly and you've highlighted all necessary dependencies for installation?
- [x] Are the software programs and packages you're highlighting in your article up to date, using current versions and not deprecated?

This closes #4585 

> Finally, delete the article checklist notes in blockquotes and submit your PR. We look forward to reviewing your article.

### CODE REVIEW

**Comment by Jethro-magaji** [comment]
I would like to review this article @ahmadmardeni1

**Comment by ahmadmardeni1** [comment]
Go ahead @Jethro-magaji

**Comment by ahmadmardeni1** [comment]
Plagiarism check done ✅

**Comment by del-ui** [comment]
@Jethro-magaji  is there any progress?

File: index.md [hunk 1]
```diff
@@ -0,0 +1,178 @@
...
+ To install, you can download your SDK manually from [here](https://developers.facebook.com/resources/audience-network-sdk-6.8.0.zip) (Not recommended) or follow the steps below to install it the right way.
+
+
+
+### Integration in Android Studio
+
+Don't have an Application? No need to worry since you can set-up an simple Application in a few steps below.
+
+#### Step 1: Create a New Project
+For those without a ready Application, open your Android Studio and create a new project with Ads activity. Since it is a test Application, you can use it without altering. This [article](https://www.section.io/engineering-education/first-android-App/) can be very useful in helping you with proper project creation.
```
Comment (Jethro-magaji) [inline index.md:83]: Please, you second person pronoun here "For those without a ready Application,". it should be "if you don't an application created already."

File: index.md [hunk 2]
```diff
@@ -0,0 +1,178 @@
...
+
+#### Step 2: Implement Facebook SDK
+Add this android dependence to the `build.gradle` file of the App module.
+```kotlin
+
+// facebook audience network SDK
+implementation 'com.facebook.android:audience-network-sdk:5.9.1'
+
+```
+This dependece integrates the SDL to your Application.
```
Comment (Jethro-magaji) [inline index.md:93]: There are spelling typos here...

**Review by Jethro-magaji — CHANGES_REQUESTED** [review]
Hello @del-ui 
Well done for coming up with such an amazing article, however, there are some grammatical errors like typos and grammar structure. I highly suggest you use [Grammarly](https://www.grammarly.com/) to edit your article for typos and grammar. Please also add an explanation below your code. Thanks.

**Comment by del-ui** [comment]
Hello @Jethro-magaji I seem to have a problem making changes to this PR since I didn't create a branch while creating the PR. Should I open a new branch with the requested changes?....Kindly

**Comment by hectorkambow** [comment]
@del-ui 

Yes - every PR should be its own distinct branch - be sure to check out our FAQ and resources for more details. 

👍

**Comment by hectorkambow** [comment]
@del-ui 
 it seems this PR is still quite a bit aways from being completed. 
 
As a rule of thumb - please only submit articles (pull requests) with a complete and fully polished article - ready to be published. 
 
All revisions and edits should be completed on your own forked repo (as to not take up room in the queue). 
 
We will be closing this and please re-submit when it is ready to be published as a new PR (and you can add the duplicate label) and ping the reviewer and myself. 

Closing a PR does NOT mean the article can not be published (or fixed) it simply means that the edits, revisions, and fixes will happen outside of the queue.
 
👍 

cc @Jethro-magaji

**Pull request closed without merging**